packages/components/src/components/popover/popover.lite.tsx

```diff
@@ -0,0 +1,53 @@
+import { onMount, Show, useMetadata, useStore } from '@builder.io/mitosis';
+import { DBPopoverState, DBPopoverProps } from './model';
+import { cls, uuid } from '../../utils';
+import { DEFAULT_ID } from '../../shared/constants';
+
+useMetadata({
+	isAttachedToShadowDom: true,
+	component: {
+		// MS Power Apps
+		includeIcon: false,
+		properties: []
+	}
+});
+
+export default function DBPopover(props: DBPopoverProps) {
+	// This is used as forwardRef
+	let component: any;
+	// jscpd:ignore-start
+	const state = useStore<DBPopoverState>({
+		_id: DEFAULT_ID,
+		handleClick: (event: any) => {
+			event.stopPropagation();
+		}
+	});
+
+	onMount(() => {
+		state._id = props.id || 'popover-' + uuid();
+		if (props.stylePath) {
+			state.stylePath = props.stylePath;
+		}
+	});
+	// jscpd:ignore-end
+
+	return (
+		<div
```

Comment:
```suggestion
		<span
```

`div` isn't possible to get used at all places / contexts, so we should switch to an inline-element like e.g. `span`.

Reply:
But even the new [popover-target](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/popover) is using a div :O

Can you tell me in detail your concern about a good old div?

Reply:
`div` is not the first choice, but the last option when choosing an element; any other element that provides semantic meaning and/or glorifies the diversity of the Hypertext Markup Language should be preferred instead of `div`.

In this special case it's about that `div` couldn't get used in any context, as it's a block-level-element, so it wouldn't be allowed in e.g. `button, whereas `span`, `i` etc. would as they are `inline`-elements.